optimade/filtertransformers/mongo.py

```diff
@@ -58,7 +58,26 @@ def not_implemented_string(self, value):
 
     def value_list(self, arg):
         # value_list: [ OPERATOR ] value ( "," [ OPERATOR ] value )*
-        raise NotImplementedError
+        # NOTE: no support for optional OPERATOR
+        values = []
+        for value in arg:
+            if value.startswith('"') and value.endswith('"'):
+                value = value[1:-1]
+            if str(value) in self.operator_map.keys():
+                raise NotImplementedError(
+                    f"OPERATOR {value} inside value_list {arg} not implemented."
+                )
+            if value.isdigit():
+                value = int(value)
+            else:
+                try:
+                    value = float(value)
+                except ValueError:
+                    pass
```

Comment:
Is there a reason for all these checks? Shouldn't it work without type-casting here, since the values in `arg` already have the proper type (hence we're also just returning `value` straight up in `def value`.